hypothesis/h
# Replace use of `app.css` from client with a pre-built bundle for pages using legacy styles

This replaces the use of the `app.css` style bundle from the client on the account settings, signup, login, 404 and email unsubscription pages with a pre-built copy of app.css, named 'legacy-site.css'.

This will allow us to [remove the remaining CSS ](https://github.com/hypothesis/client/pull/80) in the client repository which is not actually used by the client itself and also a Jinja helper for loading client CSS assets into these pages.

----

Testing:

1. Turn off 'Activity Pages' feature flag
2. Visit account settings, login, signup and some URL that 404s, check that styling looks correct
